pythnet/remote-executor/programs/remote-executor/src/state/governance_payload.rs

```diff
@@ -0,0 +1,125 @@
+use std::{ops::Deref, mem::size_of, io::ErrorKind};
+
+use anchor_lang::{prelude::*, solana_program::instruction::Instruction};
+use wormhole::Chain;
+
+use crate::{assert_or_err, error::ExecutorError};
+
+pub const MAGIC_NUMBER : u32 = 0x4d475450; // Reverse order of the solidity contract because borsh uses little endian numbers
```

Comment:
Could you add the value from the solidity contract in this comment?

Reply:
https://github.com/pyth-network/pyth-crosschain/pull/289